modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -199,16 +199,16 @@ For example, in the following `deployments` spec of an etcd Operator CSV:
 +
 [source,go]
 ----
-FROM registry.redhat.io/openshift4/ose-operator-registry:v4.2 AS builder
```

Comment:
I don't think modify it is a good idea. Anyway, we can receive this workaround this time.